client/src/actions/authActions.js

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
```

Comment:
What I had in mind when writing a ticket for setting up Redux logic is just to set up general stuff, Provider, store and some reducer and actions placeholders, you did everything related to Redux in one ticket it seems 😄